.github/workflows/conda-build-test.yml

```
@@ -0,0 +1,33 @@
+name: conda builds
+
+on:
+  push:
+    branches: [master]
+  pull_request:
```

> PR's likely to break this test are likely to be to other repos, so perhaps it should be nightly?

Nightly is a bit much, how about weekly? We can narrow down the PRs to only those that modify these files.

(I don't think we should run this too often, even if it's free to do so. Reason being we are bound to get hit by flaky builds, and it's a waste of energy for something that shouldn't change very often.)
```suggestion
  schedule: '17 22 * * 6'  # Every Saturday
  pull_request:
    paths:
      - .github/workflows/conda-build-test.yml
      - 'src/admin-guide/environments/*.ya?ml'
```